pkg/kubelet/eviction/helpers.go

```diff
@@ -708,12 +712,28 @@ func makeSignalObservations(summary *statsapi.Summary) (signalObservations, stat
 					capacity:  resource.NewQuantity(int64(*imageFs.CapacityBytes), resource.BinarySI),
 					time:      imageFs.Time,
 				}
-				if imageFs.InodesFree != nil && imageFs.Inodes != nil {
-					result[evictionapi.SignalImageFsInodesFree] = signalObservation{
-						available: resource.NewQuantity(int64(*imageFs.InodesFree), resource.DecimalSI),
-						capacity:  resource.NewQuantity(int64(*imageFs.Inodes), resource.DecimalSI),
-						time:      imageFs.Time,
-					}
+			}
+			if imageFs.InodesFree != nil && imageFs.Inodes != nil {
```

Comment:
This could be a potential bug.  

I noticed that imageFs check for inodes was nested and depending on AvailableByres and Capacity being set.  For unit test we didn't set this so I discovered this issue.  

I don't know if this would cause any actual problem in production as both should be set.  But we had different logic for nodefs so I changed this to be similar.